### PR TITLE
set default quota when no ldap attribute is defined

### PR DESCRIPTION
**Pull request by butonic**
Without this a default quota is only eventually updated. It should be updated on login.

cc @dercorn

### CODE REVIEW

**Comment by PVince81** [comment]
Please add unit tests.

**Comment by PVince81** [comment]
Any update on the unit tests ?

**Comment by PVince81** [comment]
@jvillafanez review ? (unit tests pending)

File: lib/User/User.php [hunk 1]
```diff
@@ -170,8 +170,11 @@ public function processAttributes($ldapEntry) {
 
 		//Email
 		$attr = strtolower($this->connection->ldapEmailAttribute);
+		$quotaDefault = $this->connection->ldapQuotaDefault;
 		if(isset($ldapEntry[$attr])) {
 			$this->updateEmail($ldapEntry[$attr][0]);
+		} else if (!empty($quotaDefault)) {
```
Comment (jvillafanez) [inline lib/User/User.php]: ⁉️ 

I expect to update the quota inside the quota section, not in the email section.

**Comment by PVince81** [comment]
@jvillafanez is this obsoleted by your other PR that makes all quota behavior consistent ? Or is this an addition ?

**Comment by jvillafanez** [comment]
This is obsolete from my point of view

**Comment by PVince81** [comment]
the actual quota fix will be in 9.1.5 and 9.0.9

**Comment by PVince81** [comment]
the actual quota fix will be in 9.1.5 and 9.0.9

**Pull request closed without merging**